src/main/kotlin/dev/proxyfox/pluralkt/dispatch/types/AccountLinkEvent.kt

```diff
@@ -0,0 +1,18 @@
+package dev.proxyfox.pluralkt.dispatch.types
+
+import dev.proxyfox.pluralkt.types.PkId
+import dev.proxyfox.pluralkt.types.PkReference
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonObject
+
+@Serializable
+@SerialName("LINK_ACCOUNT")
+class AccountLinkEvent : Event {
+    constructor(
+        signingToken: String,
+        systemId: PkId?,
+        referencedData: PkReference?,
+        data: JsonObject?
+    ) : super(signingToken, systemId, referencedData, data)
```

Comment:
This looks like a purely passthrough constructor; any reason to make it explicit?

Reply:
Kotlinx Serialization doesn't let primary constructors have non-variables

Reply:
Fair enough, I'll let this pass then